package/upgrade/lib.sh

```diff
@@ -210,3 +210,54 @@ detect_upgrade()
 
   UPGRADE_PREVIOUS_VERSION=$(echo "$upgrade_obj" | yq e .status.previousVersion -)
 }
+
+upgrade_addon()
+{
+  local name=$1
+  local namespace=$2
+  local version=$3
+
+  version=$(cat /usr/local/share/addons/${name}.yaml | yq e .spec.version)
+
+  cat > addon-patch.yaml <<EOF
+spec:
+  version: $version
+EOF
+
+  item_count=$(kubectl get addons.harvesterhci $name -n $namespace -o  jsonpath='{..name}')
```

Comment:
Got error and the job fail (there is `-e` flag in the script)
```
Error from server (NotFound): addons.harvesterhci.io "vm-import-controller" not found
```

I can make it work with:
```
  if ! kubectl get addons.harvesterhci vm-import-controller -n harvester-system &> /dev/null; then
     # install
  else
```

Reply:
this should now be fixed @bk201